### PR TITLE
Fuse development->fuse Prod

### PR DESCRIPTION
**Pull request by Andrew-Pohl**
Merge fuse dev to fuse production

### CODE REVIEW

**Review by leonprou — APPROVED** [review]

**Comment by leonprou** [comment]
Approving the PR.
Please make sure the version is correct (fusenet/node:nethermind-1.13.3-v.3.0.0)
And the CI/CD is run correctly

**Review by AliakseiMalyshau — APPROVED** [review]
LGTM!

Did small fix regarding version file.

File: src/Nethermind/Nethermind.JsonRpc/Modules/Eth/GasPrice/GasPriceOracle.cs [hunk 1]
```diff
@@ -71,6 +71,7 @@ public UInt256 GetGasPriceEstimate()
 
             IEnumerable<UInt256> txGasPrices = GetSortedGasPricesFromRecentBlocks(headBlock.Number);
             UInt256 gasPriceEstimate = GetGasPriceAtPercentile(txGasPrices.ToList()) ?? GetMinimumGasPrice(headBlock.BaseFeePerGas);
+            gasPriceEstimate = UInt256.Max(gasPriceEstimate, GetMinimumGasPrice(headBlock.BaseFeePerGas));
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.JsonRpc/Modules/Eth/GasPrice/GasPriceOracle.cs:74]: Let's have a flag `UseMinGasPriceInEstimates` (or better name) in config.

File: VERSION.FUSE.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+3.0.0
```
Comment (LukaszRozmej) [inline VERSION.FUSE.txt:1]: This won't be needed in Nethermind codebase?

File: src/Nethermind/Nethermind.Runner/Data/static-nodes-spark.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Runner/Data/static-nodes-spark.json:1]: please put bootnodes in chainspec

**Review by LukaszRozmej — COMMENTED** [review]
Put comments what to change before PR to Nethermind repo

File: VERSION.FUSE.txt (hunk 2 above)
Comment (Andrew-Pohl) [reply VERSION.FUSE.txt:1]: This is just use by the CICD pipeline for version control of docker container so yeah won't be in PR back into NM 😊

File: src/Nethermind/Nethermind.JsonRpc/Modules/Eth/GasPrice/GasPriceOracle.cs (hunk 1 above)
Comment (Andrew-Pohl) [reply src/Nethermind/Nethermind.JsonRpc/Modules/Eth/GasPrice/GasPriceOracle.cs:74]: Sounds good! Will make this change

**Comment by Andrew-Pohl** [comment]
> Put comments what to change before PR to Nethermind repo

TY @LukaszRozmej will sort the above

File: src/Nethermind/Nethermind.Runner/Properties/launchSettings.json [hunk 4]
```diff
@@ -277,6 +277,10 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
+    "fuse": {
+      "commandName": "Project",
+      "commandLineArgs": "--JsonRpc.Enabled true --config configs\\fuse.cfg --baseDbPath C:\\nethermind_db"
+    },
```
[the remark this answers is not in the document]
Comment (LukaszRozmej) [reply src/Nethermind/Nethermind.Runner/Properties/launchSettings.json:283]: Or if you want to add it, update it to new format :)

**Pull request merged**